### PR TITLE
HTTP-Server: Set Response Content-Type when serving static files.

### PR DESCRIPTION
**Pull request by cabelitos**
Signed-off-by: Guilherme Iscaro <guilherme.iscaro@intel.com>

### CODE REVIEW

**Comment by cabelitos** [comment]
@ceolin Could you take a look please?

File: data/jsons/dependencies.json [hunk 1]
```diff
@@ -489,6 +489,16 @@
       "dependency": "libkmod",
       "type": "pkg-config",
       "pkgname": "libkmod"
+    },
+    {
+      "dependency": "libmagic",
+      "type": "ccode",
+      "headers": [
+        "<magic.h>"
```
Comment (lpereira) [inline data/jsons/dependencies.json]: magic

Comment (cabelitos) [inline data/jsons/dependencies.json]: Oops

**Comment by ceolin** [comment]
:+1: total

**Comment by cabelitos** [comment]
merged.

**Pull request closed without merging**